logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -382,6 +382,7 @@ public long write(Queueable element) throws IOException {
                     // not trigger a checkpoint creation in itself
                     TailPage tailPage = new TailPage(this.headPage);
                     tailPage.purge();
+                    currentByteSize -= tailPage.pageIO.getCapacity();
```

Comment:
the other place in this file that does this uses the getter `getPageIO()`

For consistency, can you use the getter?

Reply:
@jordansissel sure => done :)